docs/data-sources/credential_providers.md

```diff
@@ -110,13 +110,19 @@ Read-Only:
 <a id="nestedatt--credential_providers--oauth_authorization_code"></a>
 ### Nested Schema for `credential_providers.oauth_authorization_code`
 
+Required:
+
+- `lifetime` (Number) Lifetime multiplier of the OAuth Authorization Code credentials requested by the Credential Provider.
```

Comment:
Let's just take a lifetime in seconds - replicating the UI functionality precisely is not necessary

Reply:
Sure, wanted to match UI in order prevent confusions in the front end in case user display the credential.